### PR TITLE
translating locales for pt-PT

### PR DESCRIPTION
**Pull request by davidsbatista**
adding the local in pt-PT

### CODE REVIEW

File: src/locale/bootstrap-table-pt-PT.js [hunk 1]
```diff
@@ -24,19 +24,19 @@ $.fn.bootstrapTable.locales['pt-PT'] = $.fn.bootstrapTable.locales['pt'] = {
     return `A mostrar ${pageFrom} at&eacute; ${pageTo} de ${totalRows} linhas`
   },
   formatSRPaginationPreText () {
-    return 'previous page'
+    return 'p&aacute;gina anterior'
```
Comment (UtechtDustin) [inline src/locale/bootstrap-table-pt-PT.js:27]: Why the html entites ?
`Á` is part of utf8 so i guess we dont need the html entities.

**Review by UtechtDustin — CHANGES_REQUESTED** [review]

File: src/locale/bootstrap-table-pt-PT.js (hunk 1 above)
Comment (davidsbatista) [reply src/locale/bootstrap-table-pt-PT.js:27]: saw a mix of html entities and utf8, and decided to stick with just one; do you prefer all in UTF8 instead?

Comment (UtechtDustin) [reply src/locale/bootstrap-table-pt-PT.js:27]: I dont see a reason to use html entites, without the code is more readable :-)

Comment (davidsbatista) [reply src/locale/bootstrap-table-pt-PT.js]: updated! thanks for the quick feedback :)

**Review by UtechtDustin — APPROVED** [review]

**Comment by UtechtDustin** [comment]
Thanks for your contribution 👍🏻

**Pull request merged**